components/script/script_runtime.rs

```diff
@@ -900,7 +900,7 @@ impl StreamConsumer {
 
 /// Implements the steps to compile webassembly response mentioned here
 /// <https://webassembly.github.io/spec/web-api/#compile-a-potential-webassembly-response>
-#[allow(unsafe_code)]
+#[allow(unsafe_code, non_snake_case)]
```

Comment:
I think this one can be fully snake cased; the only violation I see is the parameter name _mimeType.